### PR TITLE
Pulling in changes for 8.7, setting config, and clearing cache afterwards

### PR DESCRIPTION
**Pull request by dannylamb**
**GitHub Issues**
Resolves https://github.com/Islandora-CLAW/CLAW/issues/1110 and https://github.com/Islandora-CLAW/CLAW/issues/1095 and https://github.com/Islandora-CLAW/CLAW/issues/1144

Supersedes https://github.com/Islandora-Devops/claw-playbook/pull/108/files

Installs 8.7, clears the cache, and pulls in the latest Islandora module, which has a nice refactor to normalize how we're generating urls.

### CODE REVIEW

File: post-install.yml [hunk 1]
```diff
@@ -12,17 +12,9 @@
     # Some tomfoolery to get the demo module to install.
     # Subject to https://www.drupal.org/node/2599228
     # Should be fixed in Drupal 8.7
-    - name: Install islandora_defaults module (fail ok)
+    - name: Install islandora_defaults module
       command: "{{ drush_path }} --root {{ drupal_core_path }} -y en islandora_defaults"
       ignore_errors: yes
```
Comment (whikloj) [inline post-install.yml]: This line should be removed

**Comment by dannylamb** [comment]
Looks like Travis has been pacified on https://github.com/Islandora-CLAW/islandora/pull/144.  This is good to test.  I made content and media, and verified triplestore and fedora indexing.  Should probably take a look at a translated piece of content just to be sure I don't introduce any regressions.

**Comment by seth-shaw-unlv** [comment]
`vagrant up` 🚀

**Comment by seth-shaw-unlv** [comment]
Something went wrong. The build completed successfully but for some reason our additional media fields, for example field_media_use and field_media_of, are no longer enabled on the form display by default. 
Screen Shot 2019-06-21 at 8 17 53 AM

**Comment by seth-shaw-unlv** [comment]
They are also disabled in the display settings. Fortunately everything works once I enable them.

File: post-install.yml [hunk 2]
```diff
@@ -9,20 +9,10 @@
...
-    - name: Update entities
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y entity:updates"
-    - name: Uninstall islandora_defaults module
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y pmu islandora_defaults"
-    - name: Install islandora_defaults module (should not fail)
+
+    - name: Install islandora_defaults module
       command: "{{ drush_path }} --root {{ drupal_core_path }} -y en islandora_defaults"
-    - name: Import feature
-      command: "{{ drush_path }} --root {{ drupal_core_path }} -y fim --bundle=islandora islandora_defaults"
```
Comment (whikloj) [inline post-install.yml]: Re: @seth-shaw-unlv's comments I also got the same behaviour. I re-added this "Import feature" and everything seemed to come up hunky-dorry.

**Comment by dannylamb** [comment]
Whoops, I must have aggressively cut that out when clearing out the
install/uninstall/reinstall rigamarole. Good jorb, Danny.

**Comment by whikloj** [comment]
job != jaerb

**Comment by dannylamb** [comment]
Alright, good to go here.  Sorry about that.

**Comment by whikloj** [comment]
`vagrant up`

**Comment by whikloj** [comment]
@dannylamb I'm assuming I need to merge https://github.com/Islandora-CLAW/islandora/pull/144, then you need to update this PR, then I can merge this?

**Comment by dannylamb** [comment]
@whikloj You got it.

**Comment by dannylamb** [comment]
@whikloj Updated to point back to dev-8.x-1.x

**Pull request merged**